src/markdownbox.jl

```diff
@@ -147,6 +149,15 @@ function render_element(md::Markdown.List, l::MarkdownBox, idx)
 end
 
 
+
+function render_element(md::Markdown.HorizontalRule, l::MarkdownBox, idx)
+    ax = Axis(l.layout[idx,1], height=l.textsize)
+    hidespines!(ax)
+    hidedecorations!(ax)
+    lines!(ax, [-1,1], [0,0], color=l.divider_color)
+end
```

Comment:
Could you do this with an LScene or a Box instead?  Axis has a significant amount of overhead, I think.

One could theoretically do something like this:
```suggestion
function render_element(md::Markdown.HorizontalRule, l::MarkdownBox, idx)
    lsc = LScene(l.layout[idx, 1]; height = l.textsize, show_axis = false)
    update_cam!(lsc.scene, Makie.campixel!)
    lines!(lsc.scene, Point2f[(-1,0), (1,0)]; space = :clip)
end

Reply:
That is exactly what I wanted to do, but I did not know how to.

Thanks for your suggestion, I happily add it.

Reply:
Glad it worked!

In theory it could have been simpler with a label, but I'm not exactly sure how to get the bbox of a layoutable as an Observable.